users_test.go

```diff
@@ -105,6 +139,66 @@ func TestUserApi_GetUser(t *testing.T) {
 	expect(t, 0, u.Following, "u.Following")
 	expect(t, date("2008-01-14T04:33:35Z"), u.CreatedAt, "u.CreatedAt")
 	expect(t, date("2008-01-14T04:33:35Z"), u.UpdatedAt, "u.UpdatedAt")
+	expectNil(t, u.SuspendedAt, "u.SuspendedAt")
+}
+
+func TestUserApi_GetUserSuspended(t *testing.T) {
+	ts, api, signal := makeGitHubAPITestServer(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL != nil && r.URL.Path == "/users/octocat" {
+			b, err := ioutil.ReadAll(r.Body)
+
+			expectNil(t, err, "err")
+
+			_, err = w.Write([]byte(getUserOctocatSuspendedResponse))
+
+			expectNil(t, err, "err")
+			expect(t, "GET", r.Method, "r.Method")
+			expect(t, "", string(b), "r.Body")
+		} else {
+			w.WriteHeader(404)
+		}
+	})
+	defer ts.Close()
+
+	u, err := api.User.GetUser("octocat")
+	waitSignal(t, signal)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expect(t, "octocat", u.Login, "u.Login")
+	expect(t, 1, u.ID, "u.Id")
+	expect(t, "https://github.com/images/error/octocat_happy.gif", u.AvatarURL, "u.AvatarUrl")
+	expect(t, "", u.GravatarID, "u.GravatarId")
+	expect(t, "https://api.github.com/users/octocat", u.URL, "u.Url")
+	expect(t, "https://github.com/octocat", u.HTMLURL, "u.HtmlUrl")
+	expect(t, "https://api.github.com/users/octocat/followers", u.FollowersURL, "u.FollowersUrl")
+	expect(t, "https://api.github.com/users/octocat/following{/other_user}", u.FollowingURL, "u.FollowingUrl")
+	expect(t, "https://api.github.com/users/octocat/gists{/gist_id}", u.GistsURL, "u.GistsUrl")
+	expect(t, "https://api.github.com/users/octocat/starred{/owner}{/repo}", u.StarredURL, "u.StarredUrl")
+	expect(t, "https://api.github.com/users/octocat/subscriptions", u.SubscriptionsURL, "u.SubscriptionsUrl")
+	expect(t, "https://api.github.com/users/octocat/orgs", u.OrganizationsURL, "u.OrganizationsUrl")
+	expect(t, "https://api.github.com/users/octocat/repos", u.ReposURL, "u.ReposUrl")
+	expect(t, "https://api.github.com/users/octocat/events{/privacy}", u.EventsURL, "u.EventsUrl")
+	expect(t, "https://api.github.com/users/octocat/received_events", u.ReceivedEventsURL, "u.ReceivedEventsUrl")
+	expect(t, "User", u.Type, "u.Type")
+	expect(t, false, u.SiteAdmin, "u.SiteAdmin")
+
+	expect(t, "monalisa octocat", u.Name, "u.Name")
+	expect(t, "GitHub", u.Company, "u.Company")
+	expect(t, "https://github.com/blog", u.Blog, "u.Blog")
+	expect(t, "San Francisco", u.Location, "u.Location")
+	expect(t, "octocat@github.com", u.Email, "u.Email")
+	expect(t, false, u.Hireable, "u.Hireable")
+	expect(t, "There once was...", u.Bio, "u.Bio")
+	expect(t, 2, u.PublicRepos, "u.PublicRepos")
+	expect(t, 1, u.PublicGists, "u.PublicGists")
+	expect(t, 20, u.Followers, "u.Followers")
+	expect(t, 0, u.Following, "u.Following")
+	expect(t, date("2008-01-14T04:33:35Z"), u.CreatedAt, "u.CreatedAt")
+	expect(t, date("2008-01-14T04:33:35Z"), u.UpdatedAt, "u.UpdatedAt")
+	expect(t, date("2018-03-07T03:36:12-06:00"), u.SuspendedAt, "u.SuspendedAt")
```

Comment:
I would only test:  ``SuspendedAt`` to contain a date. 
The rest of fields are well tested in previous test methods, don't you think?